From: Emmanuel Pescosta <emmanuelpescosta099@gmail.com>
Date: Thu, 30 Apr 2015 08:12:24 +0000
Subject: Use user-places.xbel instead of bookmarks.xml in places model.
X-Git-Url: http://quickgit.kde.org/?p=kde-baseapps.git&a=commitdiff&h=270d2affaafcc9e87ba14f82bfe579e22607e2d2
---
Use user-places.xbel instead of bookmarks.xml in places model.

FIXED-IN: 15.04.1
CCBUG: 345174
---


--- a/dolphin/src/panels/places/placesitemmodel.cpp
+++ b/dolphin/src/panels/places/placesitemmodel.cpp
@@ -85,8 +85,8 @@
     Baloo::IndexerConfig config;
     m_fileIndexingEnabled = config.fileIndexingEnabled();
 #endif
-    const QString file = KStandardDirs::locateLocal("data", "kfileplaces/bookmarks.xml");
-    m_bookmarkManager = KBookmarkManager::managerForFile(file, "kfilePlaces");
+    const QString file = KStandardDirs().localxdgdatadir() + "user-places.xbel";
+    m_bookmarkManager = KBookmarkManager::managerForExternalFile(file);
 
     createSystemBookmarks();
     initializeAvailableDevices();

